healthyregions/SDOHPlace-MetadataManager
# #19: Update solrconfig.xml and schema.xml for the updated suggest mechanism

This PR addresses #19 and includes updates to solrconfig.xml and schema.xml to accommodate the new suggestion mechanism (see https://github.com/healthyregions/SDOHPlace/pull/352 for front-end testing details).

Boost is not working that well, but it should show some difference. You can check https://solr.sdohplace.org/solr/blacklight-core-dev/suggest?suggest.q=so as an example. I only take `weight>50` ones for now as the auto-complete results and the related results, but we can always change them later. 

## How to Test

Since these changes haven't been merged into the main branch, I recommend testing by recreating the blacklight-core-dev instance and overwriting the contents of /var/solr/data/blacklight-core-dev/conf/schema.xml and /var/solr/data/blacklight-core-dev/conf/solrconfig.xml. 

Then, restart the core and use the same method described in https://github.com/healthyregions/SDOHPlace/pull/352 to verify the results.
